bin/initPolykey.ts

```diff
@@ -1,10 +1,12 @@
 import fs from 'fs'
```

Comment:
Everything in `bin` should actually be a real command. I'm pretty sure `initPolykey` is not user-friendly command. I remember writing in the design repo that we should be using `pk ...` commands. Sub commands as good.

Reply:
`initPolykey` was more intended to be a function used to get a polykey instance up and running for each command called by the user. It's not exposed to the end user. It also prompts the user in case there are any things missing from polykey's config like private key or passphrase or location. We could put it in the main `polykey.ts` file perhaps?

Reply:
There is now only one file in the `bin` directory, everything else is in the `src` folder.